app/invites/server/startup/permissions.js

```diff
@@ -0,0 +1,11 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Permissions } from '../../../models';
+
+Meteor.startup(() => {
+	if (Permissions) {
+		if (!Permissions.findOne({ _id: 'create-invite-links' })) {
+			Permissions.insert({ _id: 'create-invite-links', roles: ['admin', 'moderator', 'owner'] });
```

Comment:
Why are you adding this entire file instead of list the new permission [here](https://github.com/RocketChat/Rocket.Chat/blob/accb833be634b43dbe14e6fdc97005830ec6080b/app/authorization/server/startup.js#L111)